src/main/scala/viper/server/core/VerificationWorker.scala

```diff
@@ -228,9 +226,12 @@ class ViperBackend(private val _frontend: SilFrontend, private val programId: St
         }
       }
     } else {
-      _frontend.logger.debug(s"Inconsistent error splitting; no cache update for this verification attempt with ProgramID $programId.")
+      _frontend.logger.warn(s"Inconsistent error splitting; no cache update for this verification attempt with ProgramID $programId.")
     }
 
+    // Write cache to file at the end of a verification run
+    ViperCache.writeToFile()
```

Comment:
That would cause the cache to be lost if the client forgets to shut down ViperServer or if the program is terminated unexpectedly. Would this be the desired behavior?

Reply:
Good question. We could make this configurable as well. I'm a bit concerned that this introduces a large overhead as soon as the cache starts to get bigger. I guess we will figure this out later but it's good to keep this in mind